v5/core/base_service.go

```diff
@@ -348,25 +391,9 @@ func (service *BaseService) Request(req *http.Request, result interface{}) (deta
 		}
 	}
 
+	// Invoke the request, then check for errors during the invocation.
 	var httpResponse *http.Response
-
-	// Try to get the retryable Client hidden inside service.Client
-	retryableClient := getRetryableHTTPClient(service.Client)
-	if retryableClient != nil {
-		retryableRequest, retryableErr := retryablehttp.FromRequest(req)
-		if retryableErr != nil {
-			err = fmt.Errorf(ERRORMSG_CREATE_RETRYABLE_REQ, retryableErr.Error())
```

Comment:
As I see this constant (`ERRORMSG_CREATE_RETRYABLE_REQ`) is not in use anymore, so it should be deleted, right?

Reply:
yes, good catch.  I'll delete that.